subcommands/waves/rollout.go

```diff
@@ -53,3 +72,24 @@ func readGroup(cmd *cobra.Command, args []string) string {
 	}
 	return group
 }
+
+func readUuids(cmd *cobra.Command) []string {
+	uuids, _ := cmd.Flags().GetString("uuids")
+	if len(uuids) == 0 {
+		return nil
+	} else if uuids[0] == '@' {
+		if content, err := os.ReadFile(uuids[1:]); err != nil {
+			subcommands.DieNotNil(err, "Failed to read the devices UUIDs from a file:")
+		} else if len(content) == 0 {
+			subcommands.DieNotNil(errors.New("Devices UUIDs file cannot be empty"))
+		} else {
+			uuids = string(content)
+		}
+	}
+	res := strings.Split(uuids, ",")
```

Comment:
Bikeshed warning: I wonder if this would be more natural as a line-break delimited file?

Also - probably want to strip whitespace from each item in the array. People will likely do things like: `uuid1, uuid2`.

Reply:
I planned to output the uuids as comma-separated list, and allow users to do smth like `fioctl waves rollout <wave> --limit=10 --print-uuids <file>`.
But I agree line break is better as a separator for the file, so we can be smart here.

Anyway, I _believe_ we can support both a comma and a line break here using `strings.FieldsFunc`.
Then, we can obviously trim the output and even filter out any empty space produced by e.g.
```
uuid1, uuid2,,,,,uuid3
uuid4

uuid5
```

Reply:
even better. thanks